Python/Delete-Tweets/README.md

```diff
@@ -0,0 +1,50 @@
+
+# Delete Tweets Automatically using Python
```

Comment:
I think `Delete Tweets Automatically` is good enough here.

Reply:
Done